messages/ZH.json

```diff
@@ -0,0 +1,317 @@
+      "pageCreepHtml": "在上列书刊装订流程的第 5 步中，页面已经裁切以使未装订的那一侧齐平。一叠纸中间的纸上切掉的纸，多于一叠纸表面的纸上切掉的纸，因为其边缘突出幅度更大。这称为<i>页面错动幅度</i>。为了抵消页面错动的影响，使所有页面在书刊装订并裁切完毕后出现在同一位置，必须对所有页面都稍作调整，具体调整幅度取决于纸张厚度之类的因素。对每一页都会按<strong>页面错动幅度</strong>中设置的值稍作调整。要了解这究竟有什么作用，请试着将<strong>页面错动幅度</strong>设置为半英寸或大约十毫米，然后仔细研究调整效果。"
+    }
+  },
+  "rotate": {
+    "rotation": "旋转",
+    "degrees": "角度",
+    "degreesHelp": "页面旋转角度。",
+    "help": "PDF Snake 的“旋转”工具可以任何角度旋转页面。"
+  },
+  "pageRange": {
+    "pageRange": "页面范围",
+    "help": {
+      "intro": "页面范围通常按以下格式指定：",
+      "generalExample": "first-last odd? even?",
+      "someExamples": "但是，最好的页面范围指定方法理解方式是仔细研究一些示例：",
+      "allThePages": "文档中的所有页面",
+      "pages-3-10-15": "第 3、10、15 页",
+      "pages-10-15": "第 10 到第 15 页（6 页）",
+      "pages-3-4-10-15": "第 3、4、10 页到第 15 页（8 页）",
+      "pages-10-15-even": "第 10、12、14 页",
+      "pages-10-15-odd": "第 11、13、15 页",
+      "pages-10-odd": "第 10 页之后的所有奇数页",
+      "pages-1-10-even-10-odd": "第 2、4、6、8、10 页和第 10 页之后的所有奇数页",
+      "pages-even": "所有偶数页"
+    }
+  },
+  "shuffle": {
+    "note": "注：可用 {link} 合并多个文档中的页面。",
+    "help": {
+      "intro": "PDF Snake 的“随机排序”工具可对文档中的页面进行随机排序。这是一个多合一步骤，可对页面进行重新排序，删除重复页面，添加空白页，还可以旋转页面。",
+      "format": "页面顺序一般按以下格式指定，非常类似于页面范围指定格式。",
+      "allThePages": "文档中的所有页面，顺序相同",
+      "theLastPage": "文档中的最后一页",
+      "theSecondToLastPage": "文档中的第二页到最后一页",
+      "pages-15-10-3": "第 15、10、3 页",
+      "pages-15-10": "第 10 到第 15 页（6 页），顺序相反",
+      "allReversed": "文档中的所有页面，顺序相反",
+      "pages-15-10-odd": "第 15、13、11 页",
+      "pages-odd-even": "首先是所有奇数页，其次是所有偶数页",
+      "pages-odd-1-even": "首先是所有奇数页，其次是所有偶数页，顺序相反",
+      "pages-5-1": "第一页的五个副本",
+      "pages-2-odd-even-8": "首先是所有奇数页，其次是所有偶数页，然后又是所有奇数页，然后又是所有偶数页，最后是第 8 页",
+      "pages-all-all": "第 1 页的 2 个副本，第 2 页的 2 个副本，第 3 页的 2 个副本，以此类推",
+      "pages-square-odd-even": "所有页面，顺序正常",
+      "rotateIntro": "PDF Snake 可在其对页面进行随机排序时旋转页面。要指定旋转角度，请在页码后使用以下符号：",
+      "rotate90": "顺时针旋转 90 度",
+      "rotate180": "旋转 180 度",
+      "rotate270": "顺时针旋转 270 度（逆时针旋转 90 度）",
+      "allUpsideDown": "文档中的所有页面，上下颠倒",
+      "pages-1-2-3-4": "第 1 页，第 2 页（顺时针旋转 90 度），第 3 页（上下颠倒），第 4 页（逆时针旋转 90 度）。",
+      "groupIntro": "对于同一文档，PDF Snake 可将某个随机排序模式重复许多次。  例如：",
+      "pages-group-3": "对于六页的文档，得出的页面顺序会是 3 2 1 6 5 4",
+      "blankIntro": "每当输入 X 时，PDF Snake 都会插入空页：",
+      "pages-1-X-3": "第 1 页，空页，第 3 页"
+    }
+  },
+  "pages": {
+    "zoomIn": "放大",
+    "zoomOut": "缩小",
+    "download": "下载 PDF",
+    "viewFirst": "查看前 {count} 页",
+    "viewPrevious": "查看此前的 {count} 页",
+    "pageRange": "{first}-{last}",
+    "viewNext": "查看此后的 {count} 页",
+    "viewFinal": "查看最后 {count} 页",
+    "recalculating": "正在重新计算……"
+  },
+  "home": {
+    "step": "第 {number} 步： {name}",
+    "help": "帮助",
+    "saveAs": "将这些步骤另存为：",
+    "cancel": "取消",
+    "saved": "已保存 {name}",
+    "addStep": "添加步骤",
+    "saveSteps": "保存步骤"
+  },

```

Comment:
3 days ago, I added a `saveButton`.  Could you also add that translation here?
https://github.com/surferjeff/pdfsnake-locales/commit/a115b67ff18395c6f9d3430c17e6d9b0d74f38ec